t/rose-task-run/06-app-prune-iso.t

```diff
@@ -58,7 +57,14 @@ else
     sed '/\$JOB_HOST/d; /my_task_2/d' \
         $TEST_SOURCE_DIR/$TEST_KEY_BASE.log >expected-prune.log
 fi
-file_cmp "$TEST_KEY" expected-prune.log edited-prune.log
+
+sort expected-prune.log > expected-prune.log.sorted
+sort edited-prune.log > edited-prune.log.sorted
+
+cp "expected-prune.log.sorted" ~/
+cp "edited-prune.log.sorted" ~/
```

Comment:
Why copy these to $HOME? Still debugging?

Reply:
🐆